.github/workflows/pull-request.yml

```diff
@@ -4,9 +4,135 @@ on:
   pull_request:
 
 jobs:
-  build-linux:
-    name: Build Linux
-    runs-on: ubuntu-20.04
+#  build-linux:
```

Comment:
Be sure to enable Linux and Windows test builds again, and this should be ready to go. :-)